ntbs-service/Pages/Notifications/Overview.cshtml

```diff
@@ -7,20 +7,27 @@
     ViewData["NotificationId"] = @Model.NotificationId;

```

Comment:
oops that was from a previous implementation before I used the partial view, will get rid of that!